wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -441,8 +441,7 @@ This can be done by entering this to the Git's CLI:
 
 ```prolog
 cd ~/osu-wiki
-git add .
-git commit -m "<short title>" -m "<long description>"
+git commit -am "<commit message>"
```

Comment:
The one given is a standard way. Also, you forgot to change the explanation below.

The `git commit -am "<commit message>"` can be an alternative way of doing it.

Make a note of the alternate command below the description of this command?

Like:-

Alternatively, you could also use
```prolog
cd ~/osu-wiki
git commit -am "<short title>" -m "<long description>"
```